modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/notifications/ObjectUserNotificationsHandler.java

```diff
@@ -72,13 +76,33 @@ protected String getLink(
 			}
 		}
 
+		if (GetterUtil.getBoolean(jsonObject.get("exceedsObjectEntryLimit"))) {
+			RequestBackedPortletURLFactory requestBackedPortletURLFactory =
+				RequestBackedPortletURLFactoryUtil.create(
+					serviceContext.getRequest());
+
+			return PortletURLBuilder.create(
+				requestBackedPortletURLFactory.createActionURL(
+					ConfigurationAdminPortletKeys.SYSTEM_SETTINGS)
+			).setMVCRenderCommandName(
+				"/configuration_admin/edit_configuration"
+			).setRedirect(
+				serviceContext.getCurrentURL()
+			).setParameter(
+				"factoryPid",
+				"com.liferay.object.configuration.ObjectConfiguration"
+			).setWindowState(
+				WindowState.MAXIMIZED
+			).buildString();
+		}
+
 		return PortletURLBuilder.create(
 			PortalUtil.getControlPanelPortletURL(
 				serviceContext.getRequest(), serviceContext.getScopeGroup(),
 				jsonObject.getString("portletId"), 0, 0,
 				PortletRequest.RENDER_PHASE)
 		).setMVCRenderCommandName(
-			"/object_entries/edit_object_entry"
+			"/configuration_admin/edit_configuration"
```

Comment:
Hey @thallescezar keep `"/object_entries/edit_object_entry"` here,

Thank you!